spec/controllers/authority_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe AuthorityController, type: :controller do
+  describe "GET #index" do
+    it "returns http success" do
```

Comment:
see below- the description doesn't match the expectation

Reply:
I've also edited this so it tests for both a 301, and that it redirects to the registers collection page, showing by organisations.